### DISCUSSION TITLE
Turn off Save Preivew dialog?

### DISCUSSION THREAD

**Discussion by bombillazo** [dbeaver/dbeaver#19662] [Q&A] [answered]
Hey, is there an option (or options) to turn off DBeaver popup dialogs asking to confirm to run queries or when I want to save data?

**Comment by E1izabeth** [comment, marked as answer]
Sure 
![image](https://user-images.githubusercontent.com/28875055/231259984-5e38c847-5fb2-44b8-8089-2a6607986bc9.png)

**Reply by bombillazo** [reply]
Thank you 🙏
